src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -108,6 +111,37 @@ export class Main extends Component {
     this.props.toggleMobileDisplayHidden(hidden);
   };
 
+  focusTrap = e => {
+    const buttonContainer = document.getElementById('va-nav-controls');
+    const megaMenuContainer = document.getElementById('mega-menu-mobile');
+    const focusable = [
+      ...Array.from(buttonContainer.querySelectorAll(tabbableSelectors)).filter(
+        el =>
+          el.getAttribute('tabindex') !== '-1' &&
+          el.getAttribute('hidden') === null,
+      ),
+      ...Array.from(
+        megaMenuContainer.querySelectorAll(tabbableSelectors),
+      ).filter(
+        el =>
+          el.getAttribute('tabindex') !== '-1' &&
+          el.getAttribute('hidden') === null,
+      ),
+    ];
+    const firstEl = focusable[0];
+    const lastEl = focusable[focusable.length - 1];
+
+    if (e.keyCode === 9) {
+      if (e.shiftKey && document.activeElement === firstEl) {
+        e.preventDefault();
+        lastEl.focus();
+      } else if (!e.shiftKey && document.activeElement === lastEl) {
```

Comment:
If you want you don't need the `else` if you use a `return` in the previous if statement, up to you which you think looks cleaner 🙂

Reply:
I think Typescript got me too into the habit of consistent return values, so I like the `if/else if`

Reply:
Oh! That makes sense 💪